### DISCUSSION TITLE
Sort multiplayer rooms by player count, instead of by newest created

### DISCUSSION THREAD

**Discussion by behsn** [ppy/osu#28856] [Ideas]
In lazer, rooms rarely ever get above like 10 players and I am convinced that this is one of the main reasons. Players are just too lazy to scroll down to older rooms lmao.

**Comment by SupDos** [comment]
I don't remember how stable sorts multi rooms off the top of my head, but it always felt decently sorted compared to just sorting by creation date, using that sorting method could be an option too

**Comment by peppy** [comment]
Not only sort order is crap, but the whole interface is designed without usability considerations. You have to dart your eyes left and right to see player counts.
